week1/week1.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+# To run this file do
+#		python3 week1.py
+
+##########Global Variables##########
```

Comment:
There's nothing under this header, so it's a bit confusing. I would remove it, or explain how everything in the root is global (to the module).

Global is kind of weird in python since you can access everything if you want...